build/configs/artik05x/board_metadata.txt

```diff
@@ -23,9 +23,9 @@ USBRULE_BOARD="artik"
 USBRULE_IDVENDOR="0403"
 USBRULE_IDPRODUCT="6010"
 
-BL1="bl1"
-BL2="bl2"
-KERNEL="tinyara_head"
-OTA="ota"
+BL1="bl1.bin"
+BL2="bl2.bin"
+KERNEL="tinyara_head.bin"
+OTA="ota.bin"
```

Comment:
Could you let me know why this should be moved? Please add the reason why this is necessary in the commit description.

Reply:
I added filename extension to a file name in board_metadata.txt to support various filename extensions (e.g. .elf, .hex).
For example, Esp32 board has a elf type, not bin type as TizenRT output file (tinyara.elf).

Reply:
You mean that we will flash the elf type for esp32 board?

Reply:
I checked we flash .elf file for esp32 in this link.
`https://github.com/Samsung/TizenRT/blob/master/build/configs/esp32_DevKitC/README.md`